Sources/Quick/World.swift

```diff
@@ -109,6 +109,18 @@ final internal class World: _WorldBase {
         isConfigurationFinalized = true
     }
 
+    /**
+     Returns `true` if the root example group from the given spec class has been already
+     initialized.
+
+     - parameter specClass: The QuickSpec class for which is checked for the existing root
+     example group.
+     */
```

Comment:
I'd like to tweak the doc comment block as:

```swift
    /**
     Returns `true` if the root example group for the given spec class has been already initialized.

     - parameter specClass: The QuickSpec class for which is checked for the existing root example group.
     - returns: Whether the root example group for the given spec class has been already initialized or not.
     */
```

(adjusting line length and adding `returns`)

Reply:
Hm, I tried to match the line length of the doc for `func rootExampleGroup(forSpecClass specClass: QuickSpec.Type) -> ExampleGroup` 🤷‍♂ 

I'll add the `returns` doc; it seemed to be a bit redundant to me.

Reply:
I think it's checked by default, isn't it? I see it checked.

<img width="261" alt="Screen Shot 2019-09-17 at 08 32 08" src="https://user-images.githubusercontent.com/6388510/65016924-c09c5900-d925-11e9-97f8-ddef7aec8c56.png">

Reply:
Hm then it's weird that I couldn't push to your fork. 😕

Reply:
> Hm, I tried to match the line length of the doc for `func rootExampleGroup(forSpecClass specClass: QuickSpec.Type) -> ExampleGroup` 🤷‍♂ 

I'll add SwiftLint line length rule configuration with 120 (or more).